DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgDocumentContentRepositoryIntegrationTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using DocumentDataAPI.Data;
+using DocumentDataAPI.Data.Repositories;
+using DocumentDataAPI.Models;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace DocumentDataAPITests.Data.Repositories;
+
+[Collection("DocumentDataApiIntegrationTests")]
+public class NpgDocumentContentRepositoryIntegrationTests
+{
+    private readonly NpgDbConnectionFactory _connectionFactory;
+    private readonly ILogger<NpgDocumentContentRepository> _logger;
+
+    public NpgDocumentContentRepositoryIntegrationTests()
+    {
+        _connectionFactory = new NpgDbConnectionFactory(TestHelper.DatabaseOptions.ConnectionString);
+        _logger = new Logger<NpgDocumentContentRepository>(new NullLoggerFactory());
+        TestHelper.DeployDatabaseWithTestData();
+    }
+
+    [Fact]
+    public void GetAll_ReturnsAllDocumentContents()
+    {
+        // Arrange
+        NpgDocumentContentRepository repository = new(_connectionFactory, _logger);
+
+        // Act
+        List<DocumentContentModel> result = repository.GetAll().ToList();
+
+        // Assert
+        result.Should().HaveCount(5, "because the document_contents table in the test database has 5 rows");
+    }
+
+    [Fact]
+    public void Update_UpdatesRow()
+    {
+        // Arrange
+        NpgDocumentContentRepository repository = new(_connectionFactory, _logger);
+        const string expected = "Test Content";
+
+        // Act
+        repository.Update(new DocumentContentModel(expected, 1));
+        string? actual = repository.Get(1).Content;
+
+        // Assert
+        actual.Should().Be(expected, "because the content of the document with id 1 was updated to (1, 'Test Content')");
+    }
+
+    [Fact]
+    public void DeleteWithNoForeignKeyViolation_DeletesRow()
+    {
+        // Arrange
+        NpgDocumentContentRepository repository = new(_connectionFactory, _logger);
+        DocumentContentModel toBeDeleted = repository.Get(5);
+        repository.Delete(toBeDeleted);
+
+        // Act
+        DocumentContentModel actual = repository.Get(5);
```

Comment:
DocumentContentModel bør være nullable her. Hvis den ikke crasher gør det ikke så meget, men VS vil i hvert fald klage over det med en træls squiggly line!

```suggestion
        DocumentContentModel? actual = repository.Get(5);
```